<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialDb_Dialog</class>
 <widget class="QDialog" name="MaterialDb_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Material - Database</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{	
	color: rgb(255, 255, 255);
	font: 9pt &quot;Segoe UI&quot;;
	background-color: rgb(43, 43, 43);
}
QScrollBar:vertical {
     border: 4px solid rgb(240, 240, 240);
     background: solid rgb(240, 240, 240);
     width: 15px;
     margin: 16px 0 16px 0;
 }
 QScrollBar::handle:vertical {
     background: rgb(128, 128, 128);
     min-height: 18px;
 }
 QScrollBar::add-line:vertical {
     border: 3px solid rgb(240, 240, 240);
     background: solid rgb(240, 240, 240);
     height: 14px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:vertical {
     border: 3px solid rgb(240, 240, 240);
     background: solid rgb(240, 240, 240);
     height: 14px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }
 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
     border: 3px solid rgb(240, 240, 240);
     width: 3px;
     height: 3px;
     background: rgb(240, 240, 240);
 }

 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: solid rgb(240, 240, 240);
 }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="CSection_groupBox">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(128, 128, 128);</string>
     </property>
     <property name="title">
      <string>Database</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QComboBox" name="ComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}</string>
          </property>
          <item>
           <property name="text">
            <string>Steel (ASTM)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Steel (Australia AS)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Steel (BS EN)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Steel (GB50017-2017)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Steel (JIS G 3136)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Concrete (BS8110-1997)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Concrete (GB50010-2010)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Concrete (HKCC-2013)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Aluminum (BS 8118)</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Material name:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableWidget" name="Material_tableWidget">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Properties</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableWidget" name="Properties_tableWidget">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(128, 128, 128);</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="OK_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}</string>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Cancel_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
